app/Model/User.php

```diff
@@ -34,6 +34,11 @@ class User extends Authenticatable
         'role',
     ];
 
+    /**
+     * Value paginate of row
+     */
+    const ROW_LIMIT = 10;
```

Comment:
nên define ở config thì hay hơn, tại vì `row_limit`  k liên quan tới user
đặt tên chẳng hạn là `length_page` và mô tả `number of rows per page`

Reply:
e sửa lại rồi nha a